src/Data/PQueue/Prio/Max/Internals.hs

```diff
@@ -45,3 +47,6 @@ instance Foldable Down where
 
 instance Traversable Down where
   traverse f (Down a) = Down <$> f a
+
+insertMin' :: k -> a -> MaxPQueue k a -> MaxPQueue k a
+insertMin' k a (MaxPQ q) = MaxPQ (Internals.insertMax' (Down k) a q)
```

Comment:
I don't see why this is needed, see above.

Reply:
It is; see above.